test/fail_compilation/diag8101b.d

```diff
@@ -1,14 +1,15 @@
 /*
 TEST_OUTPUT:
 ---
-fail_compilation/diag8101b.d(26): Error: none of the overloads of `foo` are callable using argument types `(double)`, candidates are:
-fail_compilation/diag8101b.d(17):        `diag8101b.S.foo(int _param_0)`
-fail_compilation/diag8101b.d(18):        `diag8101b.S.foo(int _param_0, int _param_1)`
-fail_compilation/diag8101b.d(28): Error: function diag8101b.S.bar `(int _param_0)` is not callable using argument types `(double)`
-fail_compilation/diag8101b.d(31): Error: none of the overloads of `foo` are callable using a const object, candidates are:
-fail_compilation/diag8101b.d(17):        `diag8101b.S.foo(int _param_0)`
-fail_compilation/diag8101b.d(18):        `diag8101b.S.foo(int _param_0, int _param_1)`
-fail_compilation/diag8101b.d(33): Error: mutable method `diag8101b.S.bar` is not callable using a const object
+fail_compilation/diag8101b.d(27): Error: none of the overloads of `foo` are callable using argument types `(double)`, candidates are:
```

Comment:
The term "argument types" seems odd here.  What about...

> none of the overloads of `foo` are callable parameter list `(double)`, candidates are:

This also removes the plural "types" with the singular parameter type oddity.

Reply:
Well, it's talking about arguments so "argument list" would work. But this is not part of the PR, I expect *argument types* is used elsewhere too.